src/main/java/com/synopsys/integration/alert/web/api/about/AboutActions.java

```diff
@@ -38,8 +40,15 @@ public AboutActions(AboutReader aboutReader) {
         this.aboutReader = aboutReader;
     }
 
-    public Optional<AboutModel> getAboutModel() {
-        return Optional.ofNullable(aboutReader.getAboutModel());
+    public ActionResponse<AboutModel> getAboutModel() {
+        ActionResponse<AboutModel> response;
+        Optional<AboutModel> content = aboutReader.getAboutModel();
+        if (content.isEmpty()) {
+            response = new ActionResponse<>(HttpStatus.NOT_FOUND);
+        } else {
+            response = new ActionResponse<>(HttpStatus.OK, aboutReader.getAboutModel().orElse(null));
```

Comment:
Would it be reasonable to do something like this here
`return aboutReader.getAboutModel().map(content -> new ActionResponse<>(HttpStatus.OK, content).orElse(new ActionResponse<>(HttpStatus.NOT_FOUND)`